website/docs/community/people.md

```diff
@@ -19,4 +19,17 @@ I love spending time with students, teaching and finding ways to improve the cur
 
 I currently work at [Bitdefender](https://www.bitdefender.com/) as a software engineer, part of the [Connected Home Security](https://www.bitdefender.com/smart-home/) team.
 
-Apart from computer science, I really enjoy skiing, dancing and cooking.
\ No newline at end of file
+Apart from computer science, I really enjoy skiing, dancing and cooking.
```

Comment:
Why is this line here as modified?

Reply:
Is it because @lizababu didn't end her commit with a newline?

Reply:
Yes.